server/gameconnection.py

```diff
@@ -163,10 +164,18 @@ def send_message(self, message):
         result = await self.EstablishConnection(peer)
         if not result:
             self.abort("Failed connecting to host {}".format(peer))
+
+        try:
+            await asyncio.wait_for(self.game.joinable, 60.0)
```

Comment:
This is too late, you need this wait prior to establishing the connection with the host, or this will not have the desired effect.